src/components/Landing/skills/skills.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { GatsbyImage } from 'gatsby-plugin-image';
+import styles from './Skills.module.scss';
+
+const Skills = ({ skills }) => (
+  <div className={styles['container']}>
+    <div className={styles['text']}>
+      <h3>Notre expertise</h3>
+      <p>pour rester à la pointe de la technologie</p>
+    </div>
+    {displaySkills(skills)}
+  </div>
+);
+
+const displaySkills = (skills: any[]) => {
+  const render = [];
+  skills.forEach((skill, i) => {
```

Comment:
Why are you using `array.forEach()` in this component when you used `array.map()` in the other one?